### PR TITLE
Run codespell on code

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -112,6 +112,9 @@ endif
 #           use source debugging tools like delve.
 all: bin/skopeo docs
 
+codespell:
+	codespell -S Makefile,build,buildah,buildah.spec,imgtype,copy,AUTHORS,bin,vendor,.git,go.sum,CHANGELOG.md,changelog.txt,seccomp.json,.cirrus.yml,"*.xz,*.gz,*.tar,*.tgz,*ico,*.png,*.1,*.5,*.orig,*.rej" -L fpr,uint,iff,od,ERRO -w
```
Comment (mtrmac) [inline Makefile:116]: Buildah entries? I don’t think we ship a `seccomp.json`, or any change log file.
What is `imgtype`, `copy` about?

Where does `od` come from?

File: Makefile [hunk 2]
```diff
@@ -112,6 +112,9 @@ endif
...
+
 help:
```
Comment (mtrmac) [inline Makefile:118]: (Maybe move the new rule below `help` to help readers of the Makefile a bit?)

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**